modules/geo-layers/package.json

```diff
@@ -39,7 +39,8 @@
     "@math.gl/web-mercator": "^3.2.1",
     "h3-js": "^3.6.0",
     "long": "^3.2.0",
-    "math.gl": "^3.2.1"
+    "math.gl": "^3.2.1",
+    "abortcontroller-polyfill": "^1.5.0"
```

Comment:
`AbortController` is [supported on everything except IE11](https://caniuse.com/#feat=abortcontroller). Does deck.gl support IE11?

Reply:
The tests failed without the polyfill. I'm open to suggestions if there is a structure for polyfilling whatever engine the tests use and I can get rid of this dependency.

Reply:
Hmm I'm not sure then. Xiaoji probably has a better idea.

Reply:
This polyfill should be added to the root package.json as a devDependency, since it's only needed in Node.

Reply:
How do you recommend I conditionally include it from node? Something like:
```
if (__NODE__) {
    require('abortcontroller-polyfill');
}
```

Reply:
I made an attempt, but it didn't work:
https://travis-ci.com/github/visgl/deck.gl/jobs/372367371
https://github.com/visgl/deck.gl/pull/4838/commits/1805d4e40ae6fa945fde83ed5e6aa9068fd37d7d

Reply:
Import it in test/node.js

The published module is not supposed to support node.

Reply:
You need to require the polyfill after `fetch` is polyfilled:

https://travis-ci.com/github/visgl/deck.gl/jobs/372367371#L6937